KHWE-8: Implement the about view

lubovgribiniyk opened the pull request:
![joxi_screenshot_1540834151176](https://user-images.githubusercontent.com/43422576/47668512-5bc2fd80-dbb1-11e8-9429-a71c83da9e41.png)

lubovgribiniyk committed:
Implement the about view

lubovgribiniyk committed:
Update links for github user images

lubovgribiniyk committed:
Update the template function name according to the naming convention

lubovgribiniyk committed:
Move scss import to index.scss

lubovgribiniyk committed:
Merge branch 'master' of https://github.com/ihor4568/spotty-vanilla into KHWE-8-About-view

# Conflicts:
#	.gitignore
#	src/index.scss
#	webpack.config.js

ihor4568 commented:
Please mount this component inside main component. When I click on about button inside sidebar - it should be displayed

ihor4568 commented:
Please remove lorem ipsum and add some meaningful description

lubovgribiniyk committed:
KHWE-8: Change content and styles for the About component

lubovgribiniyk committed:
KHWE-8: Show the About component after click on sidebar about button

ihor4568 commented on src/components/Main/Main.js:
```diff
@@ -24,11 +27,25 @@ export class MainComponent {
   }
 
+  addEventListeners() {
+    this.aboutLink.addEventListener(
```
please use event delegation

lubovgribiniyk replied:
fixed

ihor4568 commented on src/components/About/About.js:
```diff
@@ -0,0 +1,15 @@
+import aboutTemplate from "./About.html";
+
+export class About {
```
component name should be `AboutComponent`

lubovgribiniyk replied:
fixed

ihor4568 commented:
![image](https://user-images.githubusercontent.com/43406825/47706193-4988a400-dc31-11e8-9756-2ab02db3359f.png)

please fix build

ihor4568 commented:
![image](https://user-images.githubusercontent.com/43406825/47706233-658c4580-dc31-11e8-8372-8689c2b80c89.png)

please set max width for the text - so it will look nicer on large screens

ihor4568 commented on src/components/About/About.html:
```diff
@@ -0,0 +1,46 @@
+      Our service is completely free to use.
+    </p>
+    <h4 class="about__subtitle mdc-typography--headline6">It is developed by:</h4>
+    <div class="about__links">
```
you can use loops or foreach here

lubovgribiniyk replied:
fixed

lubovgribiniyk committed:
KHWE-8: Fix review comments

- Used loop in About template
- Set the max width for the about text
- Applied the event delegation approach to handle link clicks

lubovgribiniyk committed:
KHWE-8: Change the name for the About component

lubovgribiniyk commented:
All comments were fixed.
Build works as expected on my computer. It seems that you removed Page.scss.

ihor4568 commented on src/components/Main/Main.js:
```diff
@@ -24,11 +27,24 @@ export class MainComponent {
+  handleListClick(e) {
+    e.preventDefault();
+    if (e.target.closest(".main__about-link")) {
+      this.activeView = new AboutComponent(this.mainPoint);
```
you could initialize you component in `mountChildren` and then just call `mount` here. There is no need for active view currently

ihor4568 commented on src/components/About/About.html:
```diff
@@ -0,0 +1,22 @@
+    </p>
+    <h4 class="about__subtitle mdc-typography--headline6">It is developed by:</h4>
+    <div class="about__links">
+      <% ["WOLFRIEND", "lubovgribiniyk", "yevheniiIvanise", "pavelbaranchuk",
```
please move this array to js code

lubovgribiniyk committed:
KHWE-8: Fix review comments

- Move developers array to js file
- Change logic for rendering the About component

lubovgribiniyk committed:
Merge branch 'master' of https://github.com/ihor4568/spotty-vanilla into KHWE-8-About-view
